gotext.go

```diff
@@ -142,6 +142,30 @@ func SetLibrary(lib string) {
 	loadStorage(true)
 }
 
+// GetGlobalLocalStorage is the locale storage getter for the package configuration.
+func GetGlobalLocalStorage() *Locale {
+	globalConfig.RLock()
+	storage := globalConfig.storage
+	globalConfig.RUnlock()
+
+	return storage
+}
+
+// SetGlobalLocalStorage allows overriding the global Locale object with one built manually with NewLocale().
+// This allows then to attach to the locale Domains object in memory po or mo files (embedded or in any directory),
+// for each domain.
+// Locale library, language and domain properties will apply on default global configuration.
+// Any domain not loaded yet will use to the just in time domain loading process.
+// Note that any call to gotext.Set* or Configure will invalidate this override.
+func SetGlobalLocalStorage(storage *Locale) {
+	globalConfig.Lock()
+	globalConfig.storage = storage
+	globalConfig.library = storage.path
```

Comment:
Can you walk me through the reasoning on setting the other properties here as well.... I understand it's useful for somebody trying to use a Locale object full and ready to go, but not sure if there may be other cases where having this override could be against the purpose... 

Not having the overrides for the other properties will let a little more work on the developer side, having to set all of them manually, but it's also safer because there is no unexpected behavior and also more aligned with design patterns...